cpf.go

```diff
@@ -0,0 +1,112 @@
+package godocs
```

Comment:
this package name is more generic than what this repo represents, probably `brdocs` get more specialised

Reply:
also may rename folder `brdocuments` to `brdocs`

Reply:
Done! I changed to brdocs 😃